### PR TITLE
traj_to_gif updates

### PR DESCRIPTION
**Pull request by robinsonmt1**

### CODE REVIEW

**Comment by robinsonmt1** [comment]
- [ ] Automate povray generation: Already automated functionality in ASE.
From first look, write('atoms.pov', ...) from ase.io produces atoms.pov and atoms.ini files; png files are produced using
`renderer = write('atoms.pov', ...)
renderer.render()`
- [ ] Add optional flags for manual ase_gui rendering or automated povray rendering
- [ ] Add an optional flag for whether you want to view the ase_gui
- [ ] Change os command to prevent adding additional dependencies. Akash suggests matplotlib has .png -> .gif functionality, Pillow has .png -> .gif functionality and is already a dependency for ASE

File: carmm/analyse/povray_render.py [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+    if 'radii' not in generic_projection_settings:
+        generic_projection_settings['radii'] = 1.0
+
+    if 'colors' not in generic_projection_settings:
+        generic_projection_settings['colors'] = None
+    else:
+        for atom in atoms:
+            sym = atom.symbol
+            if sym not in generic_projection_settings['colors']:
+                generic_projection_settings['colors'][sym] = jmol_colors[atomic_numbers[sym]]
```
Comment (GaryLZW) [inline carmm/analyse/povray_render.py:50]: I am just curious. We can set the color of a certain element without replacing it with another atom, right?

**Review by GaryLZW — APPROVED** [review]
Looks good to me!

File: carmm/analyse/povray_render.py (hunk 1 above)
Comment (robinsonmt1) [reply carmm/analyse/povray_render.py:50]: Yeah, this is the code for doing that. So the choice is either to substitute one element for another with a similar radius but clearer colour (e.g. I substitute Al for Kr a lot), or you can define a dictionary of element colours that you want to change and this code that you highlighted will go through and fill out the rest of the dictionary with their default jmol colours

Comment (robinsonmt1) [reply carmm/analyse/povray_render.py:50]: Happy for me to merge the pull request?

**Pull request merged**